docs/detections/rules-ui-manage.asciidoc

```diff
@@ -158,6 +161,20 @@ NOTE: When duplicating a rule with exceptions, you can choose to duplicate the r
 * Select the *All actions* menu (*...*) on a rule, then select an action.
 * Select all the rules you want to modify, then select an action from the *Bulk actions* menu.
 * To enable or disable a single rule, switch on the rule's *Enabled* toggle.
+* To <<snooze-rule-actions,snooze>> actions for rules, click the bell icon.
+
+[float]
+[[snooze-rule-actions]]
+=== Snooze rule actions
+
+Instead of turning rules off to stop alert notifications, snooze rule actions for a defined period of time. When you snooze rule actions, the rule continues to run on its defined schedule but alerts won't trigger any actions and notifications aren't sent. 
```

Comment:
Re-applying Janeen's suggestions [here](https://github.com/elastic/security-docs/pull/3217#discussion_r1196697186).

```suggestion
Instead of turning rules off to stop alert notifications, snooze rule actions for a defined time period. When you snooze rule actions, the rule continues to run on its defined schedule, but won't perform any actions and send alert notifications. 
```